content/state_pension.md

```diff
@@ -0,0 +1,23 @@
+---
+description: The pension you get from the government is called the State Pension. Pension Wise only provides guidance on defined contribution pensions.
+---
+
+# The State Pension
+
+The pension you get from the government is called the State Pension. You get it when you reach [State Pension age](https://www.gov.uk/calculate-state-pension).
+
+^Pension Wise only provides guidance on [defined contribution](/pension-types) pensions – for help with the State Pension speak to The Pensions Advisory Service on 0300 123 1047^
+
+The most you can currently get from the basic State Pension is £115.95. You don’t normally get the State Pension automatically – you have to [claim it](https://www.gov.uk/state-pension/how-to-claim).
+
+If you reach State Pension age on or after 6 April 2016 you’ll get the [new State Pension](https://www.gov.uk/new-state-pension).
+
+## Get a State Pension statement
+
+You can get a [State Pension statement](https://www.gov.uk/state-pension-statement) to find out how much you may get and the number of qualifying years on your [National Insurance record](https://www.gov.uk/check-national-insurance-record).
+
+## Defer the State Pension
+
+You can increase your income in retirement if you don’t claim your State Pension when you reach State Pension age.This is called [deferring the State Pension](https://www.gov.uk/deferring-state-pension).
```

Comment:
Fixed in https://github.com/guidance-guarantee-programme/pension_guidance/commit/b193560712b14db50c628bfd4966428fae97c55f